packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -255,6 +255,7 @@ const Alert = ({
       isFullWidth={isFullWidth}
       isBorderless={isBorderless}
       {...a11yProps}
+      {...metaAttribute('blade-component', 'alert')}
```

Comment:
Lets store these as constants somewhere? Very easy to misspell any of these and mess up our data
```suggestion
      {...metaAttribute(MetaConstants.Component, MetaConstants.Alert)}
```

Reply:
Done.